Seminars/Homeworks/Homework_003/Task_002.py

```diff
@@ -0,0 +1,10 @@
+# Напишите программу, которая найдёт произведение пар чисел списка. Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+# Пример:
+# - [2, 3, 4, 5, 6] => [12, 15, 16];
+# - [2, 3, 5, 6] => [12, 15]
+
+lst = [2, 3, 4, 5, 6]
+l = len(lst)//2 + 1 if len(lst) % 2 != 0 else len(lst)//2
```

Comment:
```suggestion
l = (len(lst) + 1) // 2
```
Мне кажется так чуть проще.